### PR TITLE
[TASK] add hint for application context in last setup step

### PR DESCRIPTION
**Pull request by hhoechtl**
If you want to try out Neos and you just run the setup your installation
will afterwards still run in development context and you might be
disappointed of the speed. So we add a hint, that there's additional
action required in order to run Neos in production context and experience
its full speed.

Resolves: NEOS-1429

### CODE REVIEW

**Comment by skurfuerst** [comment]
:+1: by reading, looks good to me!

File: TYPO3.Neos/Classes/TYPO3/Neos/Setup/Step/FinalStep.php [hunk 1]
```diff
@@ -38,9 +39,22 @@ protected function buildForm(\TYPO3\Form\Core\Model\FormDefinition $formDefiniti
 
         $docs = $congratulations->createElement('docsLink', 'TYPO3.Setup:LinkElement');
         $docs->setLabel('Read the documentation');
-        $docs->setProperty('href', 'http://docs.typo3.org/neos/');
+        $docs->setProperty('href', 'http://neos.readthedocs.org/en/stable/');
```
Comment (aertmann) [inline TYPO3.Neos/Classes/TYPO3/Neos/Setup/Step/FinalStep.php]: This should be pushed to the 1.2 branch

**Comment by aertmann** [comment]
Changed "Addresses: " to "Resolves: ", see https://discuss.neos.io/t/commit-message-style/507 or old CGL

File: TYPO3.Neos/Classes/TYPO3/Neos/Setup/Step/FinalStep.php (hunk 1 above)
Comment (kitsunet) [inline TYPO3.Neos/Classes/TYPO3/Neos/Setup/Step/FinalStep.php]: Yep, actually this part should become a separate PR to 1.2 please :)

**Comment by kitsunet** [comment]
:+1:

**Comment by aertmann** [comment]
Took care of the setup doc link in https://github.com/neos/neos-development-collection/pull/98

Will cause a merge conflict, but the currently used link is not optimal..

**Pull request merged**